src/compat/posix/include/fnmatch.h

```diff
@@ -0,0 +1,12 @@
+/**
+ * @file
+ * @brief fnmatch.h
+ *
+ * @date 20.12.18
+ * @author Filipp Chubukov
+ */
+
```

Comment:
Add header guard to avoid double inclusion, please.

Reply:
added